test/AddOption/.exclude_tests

```diff
@@ -0,0 +1,4 @@
+# for now, thee tests showing problems with processing space-separated
```

Comment:
typo thee -> these

Reply:
changed to "the" actually - thanks for the catch (sider doesn't look at such a file, I guess)